meteor/server/api/snapshot.ts

```diff
@@ -540,6 +528,73 @@ async function restoreFromSnapshot(
 		throw new Meteor.Error(402, `Unknown snapshot type "${snapshot.snapshot.type}"`)
 	}
 }
+function snapshotIsAMOSDataDump(snapshot: Record<string, any>): boolean {
```

Comment:
Perhaps it is time to remove this handling? 
The `http://localhost:3000/api/private/ingest/studio0` endpoint was added a long time ago as a replacement for this. Although this new url doesn't have a way to use it from the ui